botocore/docs/method.py

```diff
@@ -209,6 +209,8 @@ def document_model_driven_method(section, method_name, operation_model,
         return_section.style.indent()
         return_section.style.new_line()
 
+        context = {'streaming_shape': operation_model.get_streaming_output()}
```

Comment:
I like this actually more than what I originally suggested. The only possible issue that I could see is if a streaming output shape is shared elsewhere in the output shape but is not actually a streaming output, but I am having trouble seeing this happening.